chart/Chart.yaml

```
@@ -48,3 +48,8 @@ dependencies:
     version: 4.1.2
     repository: https://openebs.github.io/dynamic-localpv-provisioner
     condition: localpv-provisioner.enabled
+  - name: cert-manager
+    version: v1.17.0
+    repository: https://charts.jetstack.io
+    alias: cert-manager
+    condition: base.cert-manager.enabled
```

Not sure why we had used base for `jaeger`, but seems since then we've used the actual name, eg: `loki-stack`.
So maybe let's use the actual name?

```suggestion
    condition: cert-manager.enabled
```